pkgs/tools/misc/flashrom/default.nix

```diff
@@ -21,8 +21,12 @@ stdenv.mkDerivation rec {
   };
 
   nativeBuildInputs = [ pkg-config installShellFiles ];
-  buildInputs = [ libftdi1 libusb1 pciutils ]
-    ++ lib.optional jlinkSupport libjaylink;
+  buildInputs = [
+    (libftdi1.override { cppSupport = false; pythonSupport = false; docSupport = false; })
```

Comment:
Normally we move this to top-level to make sure not two variants of the package are used.

Reply:
You mean this should be moved to pkgs/top-level/all-packages.nix? I looked at other packages and this is how it is done there. I'm just doing the same as others.

Reply:
Honestly, I like this way more because it's more obvious for me how the package is configured. Like I never look into all-packages.nix, just this file. This file is also referenced on search.nixos.org.

Reply:
> You mean this should be moved to pkgs/top-level/all-packages.nix?

Yes

The problem is if you where to use libftdi1 anywhere else in this file without this exact overwrite you are using two variants of the same package which is usually a bad idea.

Reply:
Ah, now i see your point

Reply:
Moved the overrides to all-packages.nix.